corehq/apps/reports/static/reports/js/case_list.js

```diff
@@ -7,10 +7,16 @@ hqDefine("reports/js/case_list", ['jquery', 'analytix/js/kissmetrix', 'hqwebapp/
                 });
                 var userTypes = initialPageData.get('user_types');
                 $(document).on('click', '#apply-filters', function () {
-                    kissAnalytics.track.event("Clicked Apply",
+                    kissAnalytics.track.event("[Case List Report] Clicked Apply",
                         {"filters": _.map($("#paramSelectorForm input[name='case_list_filter']").val().split(','),
-                            function (item) {
+                            function (item, index) {
                                 if (item.substring(0,3) === "t__") { return userTypes[item.substring(3)]; }
+                                else if (item.substring(0,3) === "u__") {
+                                    if ($("input[name='case_list_filter']").select2("data")[index].is_active)
```

Comment:
Nitpick: you might pull out a variable - either `var selector = "#paramSelectorForm input[name='case_list_filter']"` or `var $input = $("#paramSelectorForm input[name='case_list_filter']")` and use it both here and above on line 11. For someone reading this code without context, it's hard to tell if the two selectors are intentionally different.